README.md

```diff
@@ -32,7 +37,7 @@ with connector.create_scanner('tmp', ScanOptions(range=RangePrefix('User.1'))) a
         print(r.row, r.cf, r.value_bytes)
 ```
 
-__Note__. This library is a work in progress. It has been tested with Accumulo 1.9 and Python 3.8.
+__Note__. This library is a work in progress. It has been tested with Accumulo 2.1.1 and Python 3.8.
```

Comment:
Looking at the open issues against the proxy - is there anything holding up a release that works with 2.1?  I'm not sure when the last release was, and I don't see any tags at all since the proxy was forked to it's own repository.  Fwiw,  we've been running with this https://github.com/apache/accumulo-proxy/pull/81 against various accumulo 2.1.X clusters and things seem to be working as expected.